src/python/pants/backend/java/lint/google_java_format/rules.py

```diff
@@ -49,7 +48,6 @@ class GoogleJavaFormatToolLockfileSentinel(GenerateToolLockfileSentinel):
 @dataclass(frozen=True)
 class Setup:
```

Comment:
Trying to elide this class with the `JvmProcess` directly causes a graph error :|

Reply:
Sorry for the trouble there. If you want to open a ticket about that linked to #11269 feel free.